src/main/kotlin/macgram/blackjackbackend/domain/user/Player.kt

```diff
@@ -0,0 +1,83 @@
+package macgram.blackjackbackend.domain.user
+
+import macgram.blackjackbackend.domain.card.Card
+import macgram.blackjackbackend.domain.card.HoldCards
+import java.math.BigDecimal
+
+class Player : Gamer {
+    val MIN_MONEY = 0L
+    val MAX_SUM = 21L
+    val cards : HoldCards = HoldCards()
```

Comment:
class를 상속 받을 때 사용되는 콜론(콜론의 좌우가 모두 클래스)과 다르게
데이터 타입을 나타내는 콜론(콜론의 좌는 변수명, 우는 타입)은 콜론의 좌우가 다릅니다.
따라서 아래와 같이 사용하는게 컨벤션에 더 알맞을 것 같습니다~
```suggestion
    val cards: HoldCards = HoldCards()
```